test/Interop/Cxx/operators/non-member-inline.swift

```diff
@@ -151,6 +151,26 @@ OperatorsTestSuite.test("greater equal (>=)") {
   expectEqual(true, result)
 }
 
+OperatorsTestSuite.test("slash equal (/=)") {
+  var lhs = LoadableIntWrapper(value: 8)
+  let rhs = LoadableIntWrapper(value: 2)
+
+  let result = lhs /= rhs
+
+  expectEqual(lhs.value, 4)
+  expectEqual(result.value, 4)
+}
+
+OperatorsTestSuite.test("star equal (*=)") {
+  var lhs = LoadableIntWrapper(value: 8)
+  let rhs = LoadableIntWrapper(value: 2)
+
+  let result = lhs *= rhs
```

Comment:
I think it would probably be better to drop the return value for `+=`/`-=`/etc, since in Swift these operators typically return Void. This isn't critical though.